Stylesheets/components/modal.css

```diff
@@ -0,0 +1,23 @@
+.modal-title{
+ font-size: 2rem;
+}
+
+.modal-text{
+    font-size: 1.5rem;
+    margin-bottom: 0rem;
+}
+
+.modal-icon{
+    padding:0rem
+}
+
+.modal-wrapper{
+    border: 0.1rem solid var(--gray-color);
```

Comment:
for such minute sizes, its recommended to use pixels rather than rems

Reply:
okay